plugins/janus_streaming.c

```diff
@@ -4713,6 +4842,21 @@ static int janus_streaming_rtsp_connect_to_server(janus_streaming_mountpoint *mp
 			vsport = atoi(transport+strlen(";server_port="));
 			JANUS_LOG(LOG_VERB, "  -- RTSP server_port (video): %d\n", vsport);
 		}
+		const char *vssrc = strstr(curldata->buffer, ";ssrc=");
+		if(vssrc != NULL) {
+			uint32_t ssrc = strtol(vssrc+strlen(";ssrc="), NULL, 16);
```

Comment:
Why is the base 16 here? SSRCs are supposed to be numeric. Or are they advertised in hex format in RTSP?

Reply:
yes, at least per rfc2636 12.39, SSRC is written in hexadecimal in the transport header from RTSP.